.github/workflows/license_check.yml

```diff
@@ -0,0 +1,56 @@
+name: Check licenses
+
+on: 
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - README.md
+
+jobs:
+  prebuild-job:
+    name: Prebuild Job
+    uses: th2-net/.github/.github/workflows/compound-prebuild-java-dev-workflow.yml@main
+    with:
+      runsOn: 'ubuntu-latest'
+
+  check_licenses:
+    runs-on: 'ubuntu-latest'
```

Comment:
We should use value from input arguments here too

Reply:
fixed